proto/decentraland/sdk/components/text_shape.proto

```diff
@@ -4,19 +4,16 @@ package decentraland.sdk.components;
 
 import "decentraland/common/colors.proto";
 import "decentraland/sdk/components/common/id.proto";
+import "decentraland/sdk/components/common/texts.proto";
 
 option (common.ecs_component_id) = 1030;
 
 message PBTextShape {
   string text = 1;
-  // @deprecated use HiddenComponent instead https://github.com/decentraland/sdk/issues/353
-  optional bool visible = 2; // default=true
-  optional string font = 3;
-  optional float opacity = 4; // default=1.0f
+  optional common.Font font = 3; // default=F_SANS_SERIF
```

Comment:
We're on time to break the compatibility. Could we re-enumerate from sequentially every number?

Reply:
yep